apps/api/src/app/inbound-parse/inbound-parse.module.ts

```diff
@@ -0,0 +1,16 @@
+import { MiddlewareConsumer, Module, NestModule } from '@nestjs/common';
+import { SharedModule } from '../../../../webhook/src/shared/shared.module';
+import { USE_CASES } from './usecases';
+import { InboundParseQueueService } from './services/inbound-parse.queue.service';
+
+const PROVIDERS = [InboundParseQueueService];
+
+@Module({
+  imports: [SharedModule],
+  providers: [...USE_CASES, ...PROVIDERS],
+  exports: [...USE_CASES],
+  controllers: [],
+})
+export class InboundParseModule implements NestModule {
+  configure(consumer: MiddlewareConsumer): MiddlewareConsumer | void {}
+}
```

Comment:
Would suggest to move the queue listener from the constructor fo the service to an actual method and listen to the onModuleInit from NestModule